config/passport.js

```diff
@@ -0,0 +1,36 @@
+const passport = require('passport');
+const JwtStrategy = require('passport-jwt').Strategy;
+const { PRIVATE_KEY } = require('../config/apiCredentials');
+
+const cookieExtractor = (request) => {
+	var token = null;
+
+	if (request && request.headers.cookie) {
+		token = request.headers.cookie.split('=')[1];
+	}
+
+	return token;
+};
+
+let options = {};
+options.jwtFromRequest = cookieExtractor;
+options.secretOrKey = PRIVATE_KEY;
```

Comment:
When it comes to defining an object and them immediately defining its properties, I like to put them together in one declaration.

Reply:
Yes, I thought I'd fixed this. Will sort this now. :)

Reply:
Done